docs/validation/timing.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Timing
+order: 4
+---
+
+# Timing of dynamic validation
+
+In the most basic way validation will happen whenever you try to submit a form. This is how server-rendered apps without any native validation or JavaScript worked for a long time. And in fact, ember-headless-form enforces validation whenever you try to submit the form.
+
+But in addition to that, as our app is running client-side, we can do validation even earlier, for example whenever an input was changed. This is what we call dynamic validation here.
+
+By default validation will happen:
```

Comment:
Two suggestions:
1. move this section (line 12) up to below the header (line 6)
2. formatting:

```suggestion
By default validation will happen:
- **initially**, when you try to submit
- on **"revalidation"**, when a field is in an invalid state (due to a prior submission attempt), and the user changes its value (a `change` event is fired from the control)
```